src/CommunityToolkit/Xamarin.CommunityToolkit/Extensions/Navigation/NavigationExtensions.android.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms.Platform.Android;
+
+namespace Xamarin.CommunityToolkit.Extensions
+{
+	public static partial class NavigationExtensions
+	{
+		static void OnShowPopup(BasePopup popup)
+		{
+			Platform.CreateRendererWithContext(popup, ToolkitPlatform.Context);
```

Comment:
That API is deprecated since Xamarin.Forms 2.5 IIRC. Is it okay for us to depend on the deprecated API? I don't think it is going to be removed in Xamarin.Forms 5 but I wonder if there are serious side-effects to using that API.

@PureWeen @jsuarezruiz @jfversluis

Reply:
I didn't know that is obsolete. So never mind. Let's go to another approach.

Reply:
I'm going to mark this thread as resolved as we have another comment already open. If we can solve this problem we can remove the `ToolkitPlatform` init logic on android.

https://github.com/xamarin/XamarinCommunityToolkit/pull/653#discussion_r536442489